@kiva/kv-shop/src/shopError.ts

```diff
@@ -0,0 +1,45 @@
+export interface ShopErrorOptions {
```

Comment:
Doesn't need to change, but in many cases `type` has replaced `interface` in TS. Kinda weird in my opinion, coming from C# world, but it's a general trend.

Reply:
For years `interface` was all the rage, which makes sense coming from Microsoft + C# land, but a few years ago they made `type` and `interface` essentially the same. I noticed that most 3rd party libraries used `type` as well, so I just assumed that behavior. I also worked with an engineer that I remember being super pro `type`... Haha. Anyways, here's the official explanation, basically there's one key difference around extensibility.

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Reply:
Aah, nice. I think that's the guide I read years ago (with updates). I've been following the rule of thumb described there:
> If you would like a heuristic, use `interface` until you need to use features from `type`.